docs/examples/code_examples/on_skipped_request.py

```diff
@@ -0,0 +1,36 @@
+import asyncio
```

Comment:
Regarding the filename, could we add some page-related prefix? I believe other examples in that folder have that.